src/lib/seam/use-seam-client.ts

```diff
@@ -18,6 +18,10 @@ export function useSeamClient(): {
     clientSessionToken,
     ...context
   } = useSeamContext()
+
+  // eslint-disable-next-line
+  const endpoint = clientOptions?.endpoint || "https://connect.getseam.com"
```

Comment:
this is the actual fix, web components are now defining `endpoint` so adds the default. You could alternatively do this inside of seamapi/javascript

Reply:
The source of truth for the default endpoint should be in `seamapi`. I think this may be an upstream issue in seamapi, why does `endpoint: undefined` not result in the default endpoint used by the `Seam` client?

Reply:
i think this is most likely the "real issue" upstream

![image](https://github.com/seamapi/react/assets/1910070/b0d2cb4e-79c3-428c-9989-ee2ef783a5dc)

Reply:
![image](https://github.com/seamapi/react/assets/1910070/5c2db7c5-1a11-4487-9458-084965c343a2)

Reply:
Yea, I would consider this an issue with seamapi. When undefined is treated differently then "missing" it can cause surprising behavior. IMO seamapi should be updated (in a major version) to unify that behavior.

To fix the issue in this project, we should omit the option instead of redefine the default.

Reply:
See also here, this should really be `apiKey: null` (I think the types do not allow that though, hence the `undefined`).